Gemfile.lock

```diff
@@ -591,9 +606,9 @@ GEM
       unf_ext
     unf_ext (0.0.8.2)
     unicode-display_width (2.4.2)
-    vcr (6.1.0)
-    view_component (2.74.1)
-      activesupport (>= 5.0.0, < 8.0)
+    vcr (6.2.0)
```

Comment:
Looking at the VCR folder and seeing that there are no references to `VCR`, we have no tests which make use of this gem. Is there much point in having this gem until we really need it?

When it comes to testing using VCR always seems like a last resort to me when it comes to unit testing as it requires regular maintenance and does a poor job of facilitating just the "unit" which should really be the fiddling of data before its passed to the api and fiddling of the response.